spec/models/claim/advocate_claim_spec.rb

```diff
@@ -729,26 +718,30 @@
     end
 
     describe '#calculate_total' do
-      it 'calculates the fees and expenses total' do
-        expect(claim.calculate_total).to eq(152.5)
-      end
+      it { expect(claim.calculate_total).to eq(152.5) }
     end
 
     describe '#update_total' do
-      it 'updates the total' do
-        create(:expense, claim_id: claim.id, amount: 3.0)
-        create(:misc_fee, claim_id: claim.id, rate: 0.5)
-        claim.reload
-        expect(claim.total).to eq(156.00)
+      context 'when adding fees' do
+        before do
+          create(:expense, claim_id: claim.id, amount: 3.0)
+          create(:misc_fee, claim_id: claim.id, rate: 0.5)
+          claim.reload
+        end
+
+        it { expect(claim.total).to eq(156.00) }
       end
 
-      it 'updates total when expense/fee destroyed' do
-        expense = claim.expenses.first
-        fee = claim.fees.first
-        expense.destroy
-        fee.destroy
-        claim.reload
-        expect(claim.total).to eq(146.00)
+      context 'when removing fees' do
```

Comment:
`when removing fees and expenses`. Is there a difference between fees and expenses?

Reply:
Not a big difference, but it can't hurt to be more explicit so I've updated so that the context accurately describes what the test is doing 👍